packages/eas-cli/src/commands/release/view.ts

```diff
@@ -0,0 +1,171 @@
+import { getConfig } from '@expo/config';
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import Table from 'cli-table3';
+import gql from 'graphql-tag';
+import _ from 'lodash';
```

Comment:
```suggestion
import groupBy from 'lodash/groupBy';
```
(We don't import `_`)